Assets/Scripts/Spawning/ResourceSpawning/SpawnArea.cs

```diff
@@ -0,0 +1,28 @@
+using Examen.Interactables.Resource;
+using Examen.Poolsystem;
+using FishNet.Object;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Examen.Spawning.ResourceSpawning
+{
+    public class SpawnArea : NetworkBehaviour
+    {
+        [SerializeField] private List<GameObject> _spawnedResources;
```

Comment:
```suggestion
        [SerializeField] private List<GameObject> _spawnedResources = new();
```

Reply:
Doesn't have to if it is serialized and I dont want it to be cleared when I started the application